src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -49,7 +49,14 @@ public CommandResult execute() {
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
-            final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
+            final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInNameLowerCase());
+
+            List<String> tempList = new ArrayList<>();
+            tempList.addAll(keywords);
+            tempList.replaceAll(String::toLowerCase);
+            keywords.clear();
```

Comment:
I would suggest to either 
1. add a comment saying all the old keywords are cleared and new processed list of strings are treated as the keywords
OR
2. Use a different variable to store the processed keywords list

Since it is not very trivial what is happening, the reader may not get the idea that the keywords are forcefully changed. This may lead to regression erros in future, if a different user is editing the same code assuming the keywords are not changed.